docs/pages/guides/localization.mdx

```diff
@@ -327,6 +327,11 @@ export default Text;
 
 If you need to use different icons for LTR/RTL or change styles based on this setting, you can use `I18nManager.isRTL` to get the current layout direction.
```

Comment:
```suggestion
If you need to use different icons for LTR/RTL or change styles based on this setting, you can use [`I18nManager.isRTL`](https://reactnative.dev/docs/next/i18nmanager#isrtl) to get the current layout direction.
```

Kind of related to the example below but lets add the link to the React Native doc here.